src/components/layouts/App/ActionsHandler.tsx

```diff
@@ -1,13 +1,11 @@
 import { FC, useEffect, useMemo } from 'react';
-import { useRouter } from 'next/router'
+import { useRouter } from 'next/router';
 
 import auth from '../../../utils/auth';
 
 export const ActionsHandler: FC = ({ children }) => {
   const router = useRouter();
-  const query = useMemo(() => new URLSearchParams(router.asPath.split('?')[1]), [
-    router.asPath,
-  ]);
+  const query = useMemo(() => new URLSearchParams(window.location.search), []);
```

Comment:
`router.asPath` preserves the last URL before [cleaning the hash](https://github.com/Coding-Coach/find-a-mentor/blob/b5f0db30fb91fb7295c4dc948bf84dc6a320f096/src/utils/auth.js#L135-L139).
`window.location.search` is up to date